src/data/Mesh.cpp

```diff
@@ -57,8 +57,8 @@ void odc::data::Mesh::initialize( odc::io::OptionParser& i_options,
                                   Grid_XYZ* an_yz_grid
 #endif
                                 ) {
-  real taumax       = -std::numeric_limits< real >::max();
-  real taumin       = -std::numeric_limits< real >::max();
+  real taumax       = std::numeric_limits< real >::max();
+  real taumin       = -std::numeric_limits< real >::min();
```

Comment:
Uhm, one more:

```
real taumax = -std::numeric_limits< real >::max();
real taumin =  std::numeric_limits< real >::max();
```

Note that `min()` is different from "-inf", http://en.cppreference.com/w/cpp/types/numeric_limits/min
Should be `lowest()` for that, which we don't use since it requires C++11.

Reply:
Shouldn't it be the other way around i.e.:
`real taumax = std::numeric_limits< real >::max();`
`real taumin =  -std::numeric_limits< real >::max();`

Reply:
If you want it invalid as possible, no.
Positive side-effect, if you have a loop and search, e.g., for the min of `l_array`, you can directly operate on that variable:
```
real l_min = std::numeric_limits< real >::max();
for( unsigned short l_i = 0; l_i < 10; l_i++ )
  l_min = std::min( l_array[l_i], l_min );
```